test/bundler/bundler_drop.test.ts

```diff
@@ -0,0 +1,81 @@
+import { itBundled } from "./expectBundled";
+
+describe("bundler", () => {
+  itBundled("js/drop", {
```

Comment:
the convention for these names has been filename/TestName, so this would be drop/Console, then drop/ReassignKeepsConsole